services/graphService.js

```diff
@@ -58,28 +59,46 @@ GraphService.prototype.publicRankedList = function() {
 	return dfd.promise
 }
 
-GraphService.prototype.userGraphs = function(username) {
+GraphService.prototype.userGraphs = function(username, filter, offset, limit) {
```

Comment:
Agreed about control, but not about splitting this so early. Thinking about indices, "username" and "deleted" would immediately hit an index anyway. I sat down to write something to substitute `filter` but the method names looked like Cocoa names :( Edits welcome